uspv/init.go

```diff
@@ -137,7 +120,46 @@ func (s *SPVCon) Connect(remoteNode string) error {
 		return err
 	}
 	s.WBytes += uint64(n)
+	return nil
+}
+
+// Connect dials out and connects to full nodes.
+func (s *SPVCon) Connect(remoteNode string) error {
+	var err error
+	var listOfNodes []string
+	if lnutil.YupString(remoteNode) {
+		// if remoteNode is "yes" but no IP specified, use DNS seed
+		listOfNodes, err = s.GetListOfNodes()
+		if err != nil {
+			return err
+		}
+	} else { // else connect to user-specified node
+		if !strings.Contains(remoteNode, ":") {
+			remoteNode = remoteNode + ":" + s.Param.DefaultPort
+		}
+		listOfNodes = []string{remoteNode}
+	}
 
+	for len(listOfNodes) != 0 {
+		err = s.DialNode(listOfNodes)
+		if err != nil {
+			listOfNodes = listOfNodes[1:]
+			continue
+		}
+		err = s.Handshake(listOfNodes)
+		if err != nil {
+			if len(listOfNodes) == 1 { // when the list is empty, error out
```

Comment:
remember whether the DNS seed lookup was performed or not.  Users should get an error indicating the host they supplied is down, vs DNS not working.

Reply:
Also, even if it breaks, the function continues to start an incomingMessageHandler() which will won't do anything.  Instead Connect() needs to return an error.

Reply:
oh, the break there is unnecessary since there's a return above. Deleting that.